.github/scripts/torchao_model_releases/README.md

```diff
@@ -0,0 +1,80 @@
+# Scripts for torchao model release and eval
+
+Note: all commands below are run in directory: `.github/scripts/torchao_model_releases/`
+
+## Release
+### default options
+By default, we release FP8, INT4, INT8-INT4 checkpoints, with model card pre-filled with template content, that can be modified later after we have eval results.
+
+Examples:
+```
+# Note: first login with `huggingface-cli login`, the quantized model will be uploaded to
+# the logged in user
+
+# release with default quant options (FP8, INT4, INT8-INT4)
+./release.sh --model_id Qwen/Qwen3-8B
+
+# release a custom set of quant options
+./release.sh --model_id Qwen/Qwen3-8B --quants INT4 FP8
+```
+
+### AWQ-INT4
+After eval for INT4 checkpoint is done, we might found some task has a large accuracy drop compared to high precision baseline, in that case we can do a calibration for that task, with a few samples, tasks are selected from [lm-eval](https://github.com/EleutherAI/lm-eval\uation-harness/blob/main/lm_eval/tasks/README.md). You can follow [new task guide](https://github.com/EleutherAI/lm-evaluation-harness/blob/main/docs/new_task_guide.md) to add new tasks to lm-eval.
+
+Examples:
+```
+# release AWQ-INT4 model, calibrated with a specific task
+# with some calibration_limit (number of samples)
+python quantize_and_upload.py --model_id Qwen/Qwen3-8B --quant AWQ-INT4 --push_to_hub --task bbh --calibration_limit 2
+```
+
+## Eval
+After we run the release script for a model, we can found the new models in the huggingface hub page for the user, e.g. https://huggingface.co/torchao-testing, the models will have a model card that's filled in with template content, such as information about the model and eval instructions, there are a few things we need to fill in, including 1. peak memory usage, 2. latency when running model with vllm and 3. quality measurement using lm-eval.
```

Comment:
are memory and latency measures are only with vllm? what about int8-int4 models?